doc/conf.py

```diff
@@ -9,8 +13,10 @@
 
 sys.path.insert(0, os.path.abspath("."))
 import make_external_gallery
+import make_chart_style_tables
 
 make_external_gallery.make_example_gallery()
+make_chart_style_tables.make_all()
```

Comment:
If I understand correctly, there are three .rst pages that are generated on the fly, and these pages use the 70+ images added to version control with this PR.

If this is the case: do we really want to add those 70+ images to version control? What is the purpose of having them in git? Are we verifying them against the repo during generation, like with the plotting image cache, to catch regressions?

Reply:
Yes.
And not sure if there are any benefits to adding them to git, I'm not doing anything special with them at the moment. I just added them as I saw some other images were already added to the repo as well.

Reply:
Yeah, those are all part of the [image cache](https://github.com/pyvista/pyvista/blob/e77405d0b97f2beeaf167ef5bb4dbfed619668e3/tests/plotting/test_plotting.py#L82-L146). Running tests for pyvista.plotting compares the output figures (just the VTK screenshots) to a "reference" copy that is in the image cache, tracked by git. When a figure changes a bit we get a warning during testing, and when a figure changes too much an error is raised. This signals that something drastically change the output. Either the change is reasonable, in which case the corresponding cache image must be invalidated and the new image `git add`ed, or something inadvertently broke that needs fixing.

If we don't expect to break charts in subtle ways then I we probably shouldn't add them to git. On the other hand, we could do the same image caching for the _test suite_ of charts.

Reply:
All right, well the main intent to add these was for user convenience in the docs. I guess the testing is less important, as I don't really customize anything on the python side, so it would only break/change when things are modified on the VTK side (but in that case it might be handy to see a failure somewhere maybe?).

Reply:
Can you explain what you mean by "user convenience"? When the doc build is run, it will regenerate those images anyway, right? So them being in git would only have the effect of small unintentional changes showing up as diffs, which would then lead to the binary files being updated (and binary files are not great for git, but these are smaller files so that's less of a concern).

My point is that having these in git is probably less convenient for devs :) We could remove them from git and add them to .gitignore and devs wouldn't even notice they're being generated (unless someone has to touch the code that generates them).

Reply:
User convenience for the users reading the docs, i.e. they immediately see in the docs what certain line/marker styles or color schemes look like.
I'm totally open for changing the way they are generated or stored though.

Reply:
Oh yeah, definitely keep the table :) I just meant moving stuff under the hood.